assimilation_code/modules/io/direct_netcdf_mod.f90

```diff
@@ -3003,17 +3024,16 @@ end function find_start_point
 !> replace the netCDF missing_value or _FillValue with
 !> the DART missing value.
 
-subroutine set_dart_missing_value(array, domain, variable)
+subroutine set_dart_missing_value(domain, variable, array)
```

Comment:
i'm going to object to this change as risky for limited benefit.  (but feel free to downvote me here.)